lib/config/validation.js

```diff
@@ -79,6 +79,23 @@ function validateConfig(config) {
             message: `Invalid semver range for allowedVersions: \`${val}\``,
           });
         }
+      } else if(key === 'packageRules' && val[0] !== undefined) {
+        const selectorsToValidate = ['packageNames', 'packagePatterns', 'excludePackageNames', 'excludePackagePatterns'];
+        let checkPackage = false;
+
+        for(const prop in val[0]){
```

Comment:
Check that the value of packageRules is an array first too, raise an error if not